lab-steven-johnson/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-steven-johnson",
+  "version": "1.0.0",
+  "description": "A Cowsay HTTP server",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./",
+    "start": "node server.js"
+  },
+  "keywords": [
+    "cowsay",
+    "http"
+  ],
+  "author": "CF Steven Johnson",
+  "license": "MIT",
+  "dependencies": {
+    "cowsay": "^1.1.9"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
```

Comment:
thank you for including the correct dev dependencies lol. it's nice to just npm install and not worry about other commands to type in.